<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1208</width>
    <height>673</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1208</width>
    <height>673</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Algorithmes de tri</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="graphContainer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>1189</width>
      <height>449</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1181</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="layoutBarGraph"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Génération des données aléatoires</string>
    </property>
    <widget class="QLabel" name="lblNbElements">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre d'éléments :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMinValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Valeur min :</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinMinValue">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="lblMaxValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Valeur max :</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinMaxValue">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="value">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinNbValues">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGenerateRandomDatas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Générer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Algorithmes de tri</string>
    </property>
    <widget class="QLabel" name="lblChoiceAlgo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Choisir un algorithme :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAlgoChoice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLaunchAlgo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Lancer</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1208</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
